build/configs/artik053/nettest/defconfig

```diff
@@ -1079,7 +1079,17 @@ CONFIG_DISABLE_NTLM=y
 CONFIG_CA_PATH="/rom"
 CONFIG_CA_BUNDLE="/rom/curl-certificates.crt"
 # CONFIG_CA_FALLBACK is not set
-# CONFIG_SSL_SUPPORT is not set
+CONFIG_SSL_SUPPORT=y
+# CONFIG_AXTLS is not set
```

Comment:
I think CONFIG_AXTLS is not necessary. becasue TizenRT only supports mbedTLS. I think these options could confuse developers

Reply:
Shall I remove all others except MBEDTLS ? Anyway we dont need others though CURL supports them.

Reply:
Removing all others would be good.